packages/backend/src/models/repositories/user.ts

```diff
@@ -185,10 +185,14 @@ export class UserRepository extends Repository<User> {
 		if (user.avatarUrl) {
 			return user.avatarUrl;
 		} else {
-			return `${config.url}/identicon/${user.id}`;
+			return this.getIdenticonUrl(user);
 		}
 	}
 
+	public getIdenticonUrl(user: User): string {
+		return `${config.url}/identicon/${user.id}`;
+	}
+
```

Comment:
I don't see why moving the same code into a separate function is necessary? Since its only one line, at most I think a comment would make sense here.

Reply:
Used at https://github.com/misskey-dev/misskey/blob/105e6d0b619366ca4effc7c6ac2bd7ef2ee8c4e1/packages/backend/src/server/proxy/avatar.ts#L85